api/src/test/java/uk/gov/dhsc/htbhf/claimant/controller/ClaimControllerTest.java

```diff
@@ -25,18 +37,32 @@
     @InjectMocks
     ClaimController controller;
 
-    @Test
-    void shouldInvokeClaimServiceWithConvertedClaim() {
+    @ParameterizedTest
+    @MethodSource("provideStatusesForInvokeClaimService")
+    void shouldInvokeClaimServiceWithConvertedClaim(EligibilityStatus eligibilityStatus, HttpStatus httpStatus) {
         // Given
         ClaimDTO dto = aValidClaimDTO();
         Claim convertedClaim = Claim.builder().build();
-        given(converter.convert(dto)).willReturn(convertedClaim);
+        given(converter.convert(any())).willReturn(convertedClaim);
+        given(claimService.createClaim(any())).willReturn(aValidClaimantWithStatus(eligibilityStatus));
 
         // When
-        controller.newClaim(dto);
+        ResponseEntity<ClaimResponse> response =  controller.newClaim(dto);
 
         // Then
+        ClaimResponse claimResponse = aClaimResponseWithEligibilityStatus(eligibilityStatus);
+        assertThat(response).isNotNull();
+        assertThat(response.getStatusCode()).isEqualTo(httpStatus);
+        assertThat(response.getBody()).isEqualTo(claimResponse);
+        verify(converter).convert(dto);
         verify(claimService).createClaim(convertedClaim);
     }
 
+    private static Stream<Arguments> provideStatusesForInvokeClaimService() {
```

Comment:
I think these could be done in a csv source rather than a method. Down to personal preference, I guess.

Reply:
This has been updated to use `@CsvSource`